### PR TITLE
Add indexes to fix getIncognitoProfile() query perf

### PR DESCRIPTION
**Pull request by kewitz**
This is currently our most expensive query.

### CODE REVIEW

**Comment by znarf** [comment]
@kewitz can you integrate feedback and ship that?

**Comment by kewitz** [comment]
@znarf yes, will do first thing after our meeting.

File: migrations/20200514164200-add-collective-index-for-incognito-query.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // Enhance User.prototype.getIncognitoProfile() performance
+    await queryInterface.sequelize.query(`
+      CREATE INDEX IF NOT EXISTS collectives_incognito_query ON "public"."Collectives"("CreatedByUserId");
```
Comment (znarf) [inline migrations/20200514164200-add-collective-index-for-incognito-query.js:7]: I would use a more generic name for the index.

Comment (kewitz) [reply migrations/20200514164200-add-collective-index-for-incognito-query.js]: Cool.
Renamed using the standard field name for the index, I'm keeping the comment as paper-trail.

**Comment by kewitz** [comment]
Well, it looks like it is working.
image

**Pull request merged**